protocols/v2/binary-sv2/serde-sv2/src/lib.rs

```diff
@@ -72,7 +72,7 @@
 //! [rkyv1]: https://docs.rs/rkyv/0.4.3/rkyv
 //! [rkyv2]: https://davidkoloski.me/blog/rkyv-is-faster-than/
 
-#![cfg_attr(feature = "no_std", no_std)]
+#![no_std]
```

Comment:
why aren't we following this pattern here?
```
#![cfg_attr(feature = "no_std", no_std)]
```

Reply:
KISS.